e2e-tests/README.md

```diff
@@ -2,15 +2,20 @@
 
 This is an automation tests folder for Virtual Visits sample, which contains all UI tests.
 
-## Build and Run End to End tests
+## Build and Run End to End tests locally
 
-- To install supported browsers, use the following command:
+- Before running the tests make sure that the environment variable for connection string is set. [Learn more about environment variables and how to set them.](https://github.com/Azure-Samples/communication-services-virtual-visits-js#environment-variables)
+
+- To install dependencies, use the following command:
 
 ```
+npm install
 npx playwright install
 
 ```
 
+For more information about browsers, check the [Official Playwright Documentation for Browsers](https://playwright.dev/docs/browsers)
+
 - To build and run the tests, use the following commands:
 
```

Comment:
nitpick, this will make a new line in the code block.

![image](https://user-images.githubusercontent.com/94404633/175437731-bae45f32-9230-4ed8-acc7-10451546bc5e.png)